Library/Homebrew/tap.rb

```diff
@@ -482,10 +487,22 @@ def contents
     contents
   end
 
+  # Find all files under a given directory and store in cache.
+  # @private
+  def self.find_in_dir(dir)
```

Comment:
I know we don't do this enough but:

```suggestion
  private_class_method def self.find_in_dir(dir)
```

(or equivalent syntax)

Reply:
@Bo98 can't do this because we're calling from an instance method.

Reply:
Oh misread this, sorry.